src/main/resources/requests.properties

```diff
@@ -17,3 +17,104 @@ requests.query.percentage.years=SELECT year(startdate) as year, count(*) as tota
   GROUP BY year \
   ORDER BY year DESC \
   LIMIT 2
+
+######################################### DUAL TIME SERIES GRAPH PROPERTIES ###########################################
```

Comment:
If we agree on creating a new class for the properties, we can also move these properties to a separated file `dualtimeseries.properties`.